apps/studio/src/lib/db/client.ts

```diff
@@ -59,6 +59,7 @@ export interface DatabaseClient {
   alterRelationSql: (changes: RelationAlterations) => string | null
   alterRelation: (changes: RelationAlterations) => Promise<void>
 
+  getChangesSql: (changes: TableChanges) => Promise<string>,
```

Comment:
These changes look great.

one nitpick: to be consistent with the other methods where we have a x, and a xSql can we name it applyChangesSql?